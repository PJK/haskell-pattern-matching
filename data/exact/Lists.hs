module Lists where

-- length :: [Int] -> Int
-- length (x:xs) = 1 + (length xs)
-- length [] = 0