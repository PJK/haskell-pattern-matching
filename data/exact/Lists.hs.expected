{
    "tag": "AnalysisError",
    "contents": {
        "tag": "GatherError",
        "contents": "Tried to use a special name qualification, not sure what to do with that yet."
    }
}